### ISSUE TITLE
The browse() method in Opera does not work.

### ISSUE THREAD

**Issue by alorenc** [pqina/filepond#265]
The browse() method in "Opera" does not work.

```vue
<file-pond
            :name="filepondInput"
            @init="handleFilePondInit"
            :ref="pond"
            ...
            />
```

```js
// calling open browse
this.$refs['pond'].browse();
```

| Environment  | Version
|------------- | -----------
| OS              | Windows
| Browser      | Opera 56.0.3051.107

**Comment by rikschennink** [comment]
How are you calling the `browse` method?

**Comment by alorenc** [comment]
Works for every browser. With the exception of the Opera.

**Comment by rikschennink** [comment]
It seems to work here. Not sure why it doesn't work in FilePond.
https://codepen.io/rikschennink/pen/vPRdqB

**Comment by alorenc** [comment]
now I see that we have not understood ;) I'm using vue-filepond.

concept (handwritten):
```javascript
<template>
    <div>
        <file-pond
            :name="filepondInput"
            @init="handleFilePondInit"
            :ref="pond"
            />
         <div @click="openBrowsePond">test</div>
    </div>
</template>

<script>
import vueFilePond, {setOptions} from 'vue-filepond';
const FilePond = vueFilePond();
export default {
        components: {
            'file-pond': FilePond,
        },
        methods: {
            openBrowsePond() {
                this.$refs['pond'].browse();
            }
        },
}
</script>
```

It does not work for the Opera.

**Comment by rikschennink** [comment]
I’ll see if I can test this next week

**Comment by alorenc** [comment]
I am sorry for my mistake. The above example works.

I had it differently, now I see what the mistake was.
1) I ran the trigger in the parent.
2) Filepond watched it.
3) When it changed, and tried to launch the browser.
But this was already beyond the onclick event.

How I changed:
1) adding to your component filepond, addEventListener ('click, this.openBrowsePond)
2) and in it I narrowed down listening for a specific object
It now works on any browser.
